### PR TITLE
[ele] fix non-EB builds

### PR DESCRIPTION
**Pull request by HawkCorrigan**
fixes T31 Elemental Shaman for non-EB builds

### CODE REVIEW

**Comment by HawkCorrigan** [comment]
waiting for approval from @Bloodmallet mostly

**Review by Bloodmallet — APPROVED** [review]
Looks good to me. Might be good to get @navv1234 opinion about this in as well to hopefully prevent unwanted side effects for Enhancement.

**Comment by HawkCorrigan** [comment]
Given that it didn't break existing profiles for Enhancement, I will merge this now so we can get some people testing on raidbots.

**Pull request merged**
